signac/common/config.py

```diff
@@ -73,7 +55,9 @@ def read_config_file(filename, configspec=None, *args, **kwargs):
     filename : str
         The path to the file to read.
     configspec : List[str], optional
-        The key-value pairs supported in the config.
+        The key-value pairs supported in the config. If None, uses the default
+        pairs supported by signac, see :data:`signac.common.validate.cfg`
+        (Default value = None).
```

Comment:
Are `configspec, *args, **kwargs` ever used? I don't see this function being called with anything except `filename` provided. Let's remove those arguments since they're not doing anything for us.